### PR TITLE
Add register command reference

### PR DESCRIPTION
**Pull request by kpietron**

### CODE REVIEW

File: en/_mdb5/standard/web/docs/cli/reference/o.html
Comment (catchmareck) [inline en/_mdb5/standard/web/docs/cli/reference/o.html:102]: Please add `<!-- prettier-ignore -->` comment before `<mdbsnippet>` tag because Prettier will mess up the indentation here.

File: en/_mdb5/standard/web/docs/cli/reference/o.html
Comment (catchmareck) [inline en/_mdb5/standard/web/docs/cli/reference/o.html]: I think we should also point out that the user will be automatically logged in so there's no need to run `mdb login` additionally.

**Review by catchmareck — CHANGES_REQUESTED** [review]
Two little changes needed.

**Comment by smolenski-mikolaj** [comment]
@cla-bot check

**Review by catchmareck — APPROVED** [review]
LGTM

**Comment by catchmareck** [comment]
@smolenski-mikolaj I'll merge this into master once we merge the register feature itself in the CLI tool. Until then, this PR needs to wait so please don't merge it yet.

**Pull request merged**
